web/src/compose_fade.js

```diff
@@ -15,25 +15,34 @@ export function set_focused_recipient(msg_type) {
     if (msg_type === undefined) {
         compose_fade_helper.clear_focused_recipient();
     }
-
+    let topic;
+    let reply_to;
+    let to_user_ids;
+    let stream_id;
     // Construct focused_recipient as a mocked up element which has all the
     // fields of a message used by util.same_recipient()
-    const focused_recipient = {
-        type: msg_type,
-    };
+    let focused_recipient;
 
-    if (focused_recipient.type === "stream") {
-        const stream_id = compose_state.stream_id();
-        focused_recipient.topic = compose_state.topic();
+    if (msg_type === "stream") {
+        stream_id = compose_state.stream_id();
+        topic = compose_state.topic();
         if (stream_id) {
-            focused_recipient.stream_id = stream_id;
+            focused_recipient = {
```

Comment:
Is there a reason behind creating `focused_recipient` even when the stream_id is not defined. Here I have changed that ,so we only create `focused_recipient` object if there is a stream present.

Reply:
I don't think so, but will try to double-check while reviewing this.

Reply:
OK I think I agree this is probably a bug fix.